src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostTaskSearchControllerTest.java

```diff
@@ -65,8 +64,9 @@ public void should_return_a_200_with_search_results() {
 
         result.then().assertThat()
             .statusCode(HttpStatus.OK.value())
-            .body("tasks.case_data.jurisdiction", everyItem(is("IA")))
-            .body("tasks.case_data.reference", hasItems(task1.get("ccdId"), task2.get("ccdId")));
+            .body("tasks.jurisdiction", everyItem(is("IA")))
+            .body("tasks.case_id", hasItem(task.get("ccdId")))
+            .body("tasks.id", hasItem(task.get("taskId")));
```

Comment:
Can we include all standard attributes in at least one of these put calls to cover the AC for POST /task(search with info)?